b2b-sample/guardio-insurance-sample-application/util/apiDecode/settings/decodeAddUser.js

```diff
@@ -20,18 +20,33 @@ import { setUsername } from "../../../util/util/apiUtil/decodeUser";
 import callAddUser from "../../apiCall/settings/callAddUser";
 import { commonDecode } from "../../util/apiUtil/commonDecode";
 
-/**
- * 
- * @param session 
- * @param firstName 
- * @param familyName 
- * @param email 
- * @param username 
- * @param password 
- * @returns `res` (if user added successfully) or `null` (if user addition was not completed)
- */
-export default async function decodeAddUser(session, firstName, familyName, email, username, password) {
-    const addUserEncode = {
+const InviteConst = {
+    INVITE: "pwd-method-invite",
+    PWD: "pwd-method-pwd"
+}
+
+
```

Comment:
Let's remove these unwanted new lines

Reply:
These have been fixed when this file was moved to a different location

### File
- https://github.com/wso2/samples-is/blob/master/b2b-sample/libs/business-admin-app/data-access/data-access-controller/src/lib/controller/user/controllerAddUser/controllerDecodeAddUser.ts